apps/apps.humanatlas.io/src/app/pages/us1/cell-population-predictions/cell-population-predictions.component.html

```diff
@@ -0,0 +1,95 @@
+<h1>Cell Population Predictions</h1>
+
+<p class="info-text">
+  This table lists all cell types predicted for the selected extraction site per tool. Note that different annotation
+  tools predict different cell types and different percentages for the very same cell type. Hover over column headers
+  for details.
+</p>
+
+<hra-workflow-card title="Cell Population Predictions" class="table-container">
+  <hra-workflow-card-actions>
+    <button mat-icon-button aria-label="more icon button with a menu" aria-hidden="false" [matMenuTriggerFor]="options">
```

Comment:
Update aria-label to `Open table action menu`
Also I don't think aria-hidden is necessary